user-manual/import-export/csv-import.rst

```diff
@@ -3100,8 +3100,7 @@ Import new authority record relationships via CSV
 .. image:: images/csv-import-page-relationships.*
    :align: center
    :width: 85%
-   :alt: The CSV import page in AtoM, set to import new Authority records
-   relationships
+   :alt: The CSV import page in AtoM, set to import new Authority records relationships
```

Comment:
Looks good. If you ever need to add a longer `alt` description than the 80-char default width, you just have to space the second line to match the first, like so:

```
:alt: The CSV import page in AtoM, set to import new Authority records
      relationships
```